src/server.js

```diff
@@ -30,9 +57,64 @@ export function makeServer({ environment = "development" } = {}) {
       this.get("/config/projects", (schema) => schema.projects.all());
       this.post("/config/projects/:projectId", () => respondOk());
       this.delete("/projects/:projectId", () => respondOk());
+      this.delete("/config/projects/:projectId", () => respondOk());
 
       // Database endpoints
       this.post("/config/projects/:projectId/database/:db/config/database-config", () => respondOk());
+      this.post("/config/projects/:projectId/database/:dbName/collections/:colName/schema/mutate", () => respondOk());
+      this.post("/config/projects/:projectId/database/:dbName/schema/mutate", () => respondOk());
+      this.post("/config/projects/:projectId/database/:dbName/collections/:colName/rules", () => respondOk());
+      this.post("/config/projects/:projectId/database/:dbName/schema/inspect", () => respondOk());
+      this.delete("/config/projects/:projectId/database/:dbName/collections/:colName", () => respondOk());
+      this.delete("/config/projects/:projectId/database/:dbName/config/database-config", () => respondOk());
+
+      // Collection endpoints
+      this.get("/external/projects/:projectId/database/:dbName/list-collections", () => respondOk());
+      this.post("/config/projects/:projectId/eventing/triggers/:triggerName", () => respondOk())
+      this.delete("/config/projects/:projectId/eventing/triggers/:triggerName", () => respondOk())
+
+      // Eventing endpoints
+      this.post("/config/projects/:projectId/eventing/config/eventing-config", () => respondOk());
+      this.post("/config/projects/:projectId/eventing/rules/:type", () => respondOk());
+      this.post("/config/projects/:projectId/eventing/schema/:type", () => respondOk());
+      this.post("/api/:projectId/graphql", () => respondOk());
```

Comment:
If the grapqhl query in the body contains `event_logs` then return event logs from the fixtures